azure-quantum/azure/quantum/qiskit/results/resource_estimator.py

```diff
@@ -1,332 +1,37 @@
-try:
-    from qiskit.result import Result
-    import markdown
-    import copy
-    from qiskit.result.models import ExperimentResult
-    from qiskit.qobj import QobjHeader
-except ImportError:
-    raise ImportError(
-        "Missing optional 'qiskit' dependencies. \
-To install run: pip install azure-quantum[qiskit]"
-    )
-
-class ResourceEstimatorBatchResult(Result):
+#!/bin/env python
+# -*- coding: utf-8 -*-
+##
+# resource_estimator.py: Qiskit result for microsoft.estimator target.
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+from azure.quantum.target.microsoft.result import MicrosoftEstimatorResult
+
+
+class ResourceEstimatorResult(MicrosoftEstimatorResult):
```

Comment:
My concern here is that we are wrapping a specialized `MicrosoftEstimatorResult` into a generic `ResourceEstimatorResult`. 

If we really need this wrapper class, then it should be named something like `MicrosoftResourceEstimatorResult`, but maybe we can just re-use the existing `MicrosoftEstimatorResult` class instead.

Reply:
The reason for this wrapper class is that `from_dict` class method that takes Qiskit data and extracts the resource estimation data from it. I don't want to add this function into the `MicrosoftEstimatorResult` since it should not have any Qiskit specific code.

I can just have a simple function that creates a `MicrosoftEstimatorResult`.